{{cookiecutter.project_name}}/.mlops-setup-scripts/README.md

```diff
@@ -190,27 +207,36 @@ python .mlops-setup-scripts/cicd/bootstrap.py \
 {%- endif %}
 ```
 
-Take care to run the Terraform bootstrap script before the CI/CD bootstrap script. This will:
+Take care to run the Terraform bootstrap script before the CI/CD bootstrap script. 
+
+The first Terraform bootstrap script will:
 
 {% if cookiecutter.cloud == "azure" %}
 1. Create an Azure Blob Storage container for storing ML resource config (job, MLflow experiment, etc) state for the
    current ML project
 2. Create another Azure Blob Storage container for storing the state of CI/CD principals provisioned for the current
    ML project
+   
+The second CI/CD bootstrap script will:
 3. Write credentials for accessing the container in (1) to a file
 4. Create Databricks service principals configured for CI/CD, write their credentials to a file, and store their
    state in the Azure Blob Storage container created in (2).
 {% if cookiecutter.cicd_platform == "azureDevOpsServices" %}
-5. Create two Azure DevOps Pipelines:
-    * `testing_ci` - Unit tests and integration tests triggered upon PR to the main branch.
-    * `terraform_cicd` - Continuous integration for Terraform triggered upon a PR to main and changes to `databricks-config`, 
-                         followed by continuous deployment of changes upon successfully merging into main.
+5. Create the two following Azure DevOps Pipelines along with required variable group:
+    * `testing_ci` - Unit tests and integration tests triggered upon PR to the {{cookiecutter.default_branch}} branch.
+    * `terraform_cicd` - Continuous integration for Terraform triggered upon a PR to {{cookiecutter.default_branch}} and changes to `databricks-config`, 
+                         followed by continuous deployment of changes upon successfully merging into {{cookiecutter.default_branch}}.
+6. Create build validation policies defining requirements when PRs are submitted to the default branch of your repository.        
```

Comment:
Here is how this is rendered. Can you fix?
<img width="1127" alt="Screen Shot 2022-12-06 at 6 03 44 PM" src="https://user-images.githubusercontent.com/66143562/206069620-6b05aa71-30da-4c4c-994f-3bf0bcc4c1ca.png">